Probe/send

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+#
+# Send messages to FCM 
+
+send_time=""
+device=""
+auth=""
+
+while [[ $1 != "" ]]; do
+  case $1 in
+    -t|--time)
+      send_time=$2
+      shift 2
```

Comment:
I gather that this is some sort of command line arg parsing, but I don't understand it.

Reply:
addressed in 765a7ee